src/ui/components/results/alternativeverticalscomponent.js

```diff
@@ -150,4 +163,36 @@ export default class AlternativeVerticalsComponent extends Component {
 
     return verticals;
   }
+
+  /**
+   * Adds parameters that are dynamically set. Removes parameters for facets,
+   * filters, and pagination, which should not persist across the experience.
+   * @param {string} baseUrl The url append the appropriate params to. Note:
+   *    params already on the baseUrl will be stripped
+   * @param {SearchParams} params The parameters to include in the experience URL
+   * @return {string} The formatted experience URL with appropriate query params
+   */
+  _getUniversalURL (baseUrl, params) {
```

Comment:
This is copied from vertical results. I believe this a not-fun example of having to copy these transformations in a few places (context/referrerPageUrl/filtering params).

This exists in a few places: VerticalResults, Navigation, AV now. I didn't refactor into its own function because of this dependency on the `this.componentManager`. There is also a weird inconsistency in the lib where `tabOrder` is added in all those places except for the universal results for both VR and AV. So what goes into this new "`getExperienceLink`" function wouldn't include tabOrder.

LMK if you want this made into another function to DRY up, I didn't know if this was in scope for a hotfix or if we wanted to get it into develop instead later (as it would mean changing navigationcomponent code outside of this fix).

Reply:
I think it's best to DRY this and put it in urlutils. We can get around the this.componentManager dependency with a bind, and we can make a second function that includes tabOrder.

Reply:
maybe the shared method could also just take in getComponentNamesForComponentTypes as a param? (we're already passing it as a param into filterParamsForExperienceUrl anyways)

Reply:
This will be addressed in SLAP-570